diff --git a/themes/mbriggs/mbriggs.theme.bash b/themes/mbriggs/mbriggs.theme.bash
index 169f8d7..621dff4 100644
--- a/themes/mbriggs/mbriggs.theme.bash
+++ b/themes/mbriggs/mbriggs.theme.bash
@@ -16,9 +16,16 @@ function git_short_sha() {
   SHA=$(git rev-parse --short HEAD 2> /dev/null) && echo "$GIT_SHA_PREFIX$SHA$GIT_SHA_SUFFIX"
 }
 
+function node_version_prompt() {
+  if which node &> /dev/null; then
+    echo -e "[$(node -v)]"
+  fi
+}
+
 function prompt() {
     local return_status=""
     local ruby="${red}$(ruby_version_prompt)${reset_color}"
+    local node="${yellow}$(node_version_prompt)${reset_color}"
     local user_host="${green}\h${reset_color}"
     local current_path="\w"
     local n_commands="\!"
@@ -26,9 +33,9 @@ function prompt() {
     local prompt_symbol='λ'
     local open='('
     local close=')'
-    local prompt_char=' \$ '
+    local prompt_char='\$ '
 
-    PS1="\n${n_commands} ${user_host} ${prompt_symbol} ${ruby} ${open}${current_path}${git_branch}${close}${return_status}\n${prompt_char}"
+    PS1="\n${n_commands} ${user_host} ${prompt_symbol} ${ruby} ${node} ${open}${current_path}${git_branch}${close}${return_status}\n${prompt_char}"
 }
 
-safe_append_prompt_command prompt
\ No newline at end of file
+safe_append_prompt_command prompt
